test/syntax.js

```diff
@@ -57,63 +54,6 @@ describe('Azure swagger schema validation:', function () {
     });
   }).value();
 
-  describe('Azure composite swagger schema validation:', function () {
```

Comment:
No syntax validation on md file, so it should be fine. 
If there's dead code I would remove it to avoid confusion.